modules/accel_app/__init__.py

```diff
@@ -2,7 +2,7 @@
 import tidal
 import accelerometer
 
-class Accel(TextApp):
+class Accel(app.TextApp):
```

Comment:
TextApp was explicitly imported from app, so does this actually work? If you were wanting to make it consistent can you update the import as well?

Reply:
Sorry, don't know how that crept in. Have pushed a fixup commit.